NitroxServer/GameLogic/Unlockables/StoryGoalData.cs

```diff
@@ -26,8 +26,12 @@ public bool RemovedLatestRadioMessage()
             {
                 return false;
             }
-
+            string message = RadioQueue[0];
             RadioQueue.RemoveAt(0);
```

Comment:
Is it possible to use our custom ThreadSafeQueue implementation for this property?